manager/orchestrator/taskreaper/task_reaper_test.go

```diff
@@ -123,6 +123,17 @@ func TestTaskReaperInit(t *testing.T) {
 		ServiceID: "goneservice",
 	}
 
+	// this third task was never assigned, and should be removed
+	terminaltask3 := &api.Task{
+		ID:           "terminaltask3",
+		Slot:         3,
+		DesiredState: api.TaskStateRemove,
+		Status: api.TaskStatus{
+			State: api.TaskStatePending,
```

Comment:
Please also add a test for TaskStateNew

Reply:
Will do. Regarding the comments below, I'll also rebase and add some more tests for the case of a running task reaper.